cds_ils/utils.py

```diff
@@ -16,3 +16,32 @@ def format_login_required_urls(urls):
             url["login_required_url"] = current_app.config[
                 "CDS_ILS_EZPROXY_URL"
             ].format(url=url["value"])
+
+
+def dynamic_delete_field(original_element, fields):
+    """Dynamically delete an array of fields from a dictionary."""
+    # Wrong parameter in the configuration
+    wrong_field = False
+
+    for field in fields:
+        if not isinstance(field, list):
+            original_element.pop(field)
+            continue
+        if not field:
+            continue
```

Comment:
This allows to define in the configuration [field1, []] (an empty array) without getting an error. An empty array should never be used regardless.

Reply:
why would we define an empty list ? If the configuration is wrong it **should** throw an error